### DISCUSSION TITLE
Follow playlist and get artist/albums

### DISCUSSION THREAD

**Discussion by HeikoGr** [Thyraz/Sonos-Kids-Controller#36] [Ideas]
a nice feature would be to follow a playlist which is splitted to artist/albums on the player. 
the administration would be way easier as you could fill the Sonos-Kids-Controller by managing an playlist on PC...

**Comment by imhofroger** [comment]
You can. Open your IP from SonosKidsController in Browser and add the Port.
http://[IP from controller]:8200
http://192.168.1.99:8200

**Reply by HeikoGr** [reply]
I know.
My idea is to have a "Sonos-Kids-Controller" Playlist in Spotify. This would be the playe to add new audibooks etc. and they are added automatically (if you understand what I mean).
It would be a nice to have feature and is absolutely not necessary in first hand.
